app/models/profile.rb

```diff
@@ -34,30 +37,16 @@ def self.attributes
     (stored_attributes[:data] || []).map(&:to_s)
   end
 
-  # Forces a reload before returning attributes
-  def self.attributes!
-    refresh_attributes!
-    attributes
-  end
-
-  # NOTE: @citizen428 This is a temporary mapping so we don't break DEV during
-  # profile migration/generalization work.
-  def self.mapped_attributes
-    attributes!.map { |attribute| MAPPED_ATTRIBUTES.fetch(attribute, attribute).to_s }
-  end
-
-  # NOTE: @citizen428 We want to have a current list of profile attributes the
-  # moment the application loads. However, doing this unconditionally fails if
-  # the profiles table doesn't exist yet (e.g. when running bin/setup in a new
-  # clone). I wish Rails had a hook for code to run after the app started, but
-  # for now this is the best I can come up with.
-  refresh_attributes! if ApplicationRecord.connection.table_exists?("profiles")
-
   def custom_profile_attributes
     custom_profile_fields.pluck(:attribute_name)
   end
 
   def clear!
     update(data: {})
   end
+
+  # NOTE: @citizen428 We want to have a current list of profile attributes the
+  # moment the application loads. I wish Rails had a hook for code to run after
+  # the app started...
```

Comment:
@citizen428 it does: `config.after_initialize`: https://guides.rubyonrails.org/configuring.html#rails-general-configuration

Reply:
I'm aware of that, but it didn't seem to work for this specific case. I may try it again  in the future, but I was just happy to have it run at one point.